exporter/datadogexporter/internal/logs/translator.go

```diff
@@ -123,11 +143,9 @@ func Transform(lr plog.LogRecord, res pcommon.Resource) datadogV2.HTTPLogItem {
 		l.Message = lr.Body().AsString()
 	}
 
-	var tags = attributes.TagsFromAttributes(res.Attributes())
-	if len(tags) > 0 {
-		tagStr := strings.Join(tags, ",")
-		l.Ddtags = datadog.PtrString(tagStr)
-	}
+	var tags = append(attributes.TagsFromAttributes(res.Attributes()), "otel:true")
+	tagStr := strings.Join(tags, ",")
+	l.Ddtags = datadog.PtrString(tagStr)
```

Comment:
we need to set this , only if it not already set

Reply:
Interestingly the reason the test `append_tags` is passing because the `ddtags` from additional properties is used later in the method.  https://github.com/DataDog/datadog-api-client-go/blob/92a9c3eb71121aa596b15b3f1e8099d3c1c0a4c5/api/datadogV2/model_http_log_item.go#L208 . 
If the order changes in the marshal method , this would fail. we should explicitly set this only once.